examples/deploy_tasks.yaml

```diff
@@ -4,30 +4,30 @@ metadata:
   name: deploy-with-helm
   namespace: default
 spec:
-    inputs:
-        resources:
-              - name: workspace
-                type: git
-              - name: imageToDeploy
-                type: image
-        params:
-            - name: pathToHelmCharts
-              description: Path to where the Helm charts live
-            - name: helmArgs
-              description: Extra arguments to pass to Helm
-        clusters:
-            - name: targetCluster
-    buildSpec:
-        steps:
-        - name: deploy
-          image: kubernetes-helm
-          command: ['helm']
-          args:
-            - 'install'
-            - '--kube-context=${inputs.clusters.targetCluster}'
-            - '--set image=${inputs.resources.imageToDeploy.url}@${inputs.resources.imageToDeploy.digest}'
-            - '${inputs.params.helmArgs}'
-            - '${inputs.params.pathToHelmChart}'
+  inputs:
+    resources:
+    - name: workspace
+      type: git
+    - name: imageToDeploy
+      type: image
+    - name: targetCluster
+      type: cluster  
+    params:
+    - name: pathToHelmCharts
+      description: Path to where the Helm charts live
+    - name: helmArgs
+      description: Extra arguments to pass to Helm
+  buildSpec:
+    steps:
+    - name: deploy
+      image: kubernetes-helm
+      command: ['helm']
+      args:
+        - 'install'
+        - '--kube-context=${inputs.resources.targetCluster.clusterName}'
```

Comment:
i think the helm command will also need to know the path to the kubeconfig file that was generated for the cluster?

Reply:
will add